config/config.go

```diff
@@ -410,6 +410,8 @@ func parseConfigString(configString string, terragruntOptions *options.Terragrun
 	terragruntOptions.ImportVariablesMap(config.Inputs, options.ConfigVarFile)
 	terragruntOptions.Logger.Tracef("Loaded configuration\n%v", color.GreenString(fmt.Sprint(terragruntConfigFile)))
 
+	config.initializeHooks()
```

Comment:
Have you considered moving that call within the `convertToTerragruntConfig` method instead ? I have the impression this adds a second call to complete the file to config convert process that was started and almost finished by the first one